src/mocks/BigMap.sol

```diff
@@ -0,0 +1,32 @@
+// Copyright 2025, Offchain Labs, Inc.
+// For license information, see:
+// https://github.com/OffchainLabs/nitro/blob/master/LICENSE.md
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.0;
+
+contract BigMap {
+    mapping(uint256 => uint256) public data;
+		uint256 size;
+
+    function storeValues(uint256 count) external {
```

Comment:
Not that it really matters, but could we just use clearAndAddValues with clear=0 instead of having this function?

Reply:
Sure. Will do.

How does one format a Solidity contract?

Reply:
In the contracts repo you run `yarn prettier:solidity`

Reply:
Fixed the formatting and made it just a single method.